### DISCUSSION TITLE
Chem.Draw.SimilarityMaps.GetSimilarityMapFromWeights - colors="k" ... what is "k"?

### DISCUSSION THREAD

**Discussion by aebrer** [rdkit/rdkit#3964] [Q&A]
I am using this function and it's fantastic! My only issue is that the default colour scheme seems to be a diverging red to green color scheme, which is problematic for a few colorblind people I work with. I'd love to change it to something else, like blue-green, or purple-green... but I don't know how! I can't find any documentation about this `colors` variable. It expects a string, but so far changing it seems to have no effect?

Does anyone know how I can change this colour scheme?

thanks :)

**Comment by aebrer** [comment]
I have tried changing `colorMap` to a few things, but none seem to make a difference.

**Comment by aebrer** [comment]
`colorMap=cm.get_cmap('PRGn')` for example does not seem to do anything. Starting to wonder if I am doing something really dumb.

**Comment by FreddieM007** [comment]
Although the question is several years old, here are some pointers in case others have the same question:

1. The default colorMap can be changed by setting it to a matplotlib string, e.g.,:
`colorMap='jet'` (or 'hsv', 'bwr', etc.)

2. Complete list of available colorMaps:
`matplotlib.pyplot.colormaps()`

3. Visual of all available colorMaps:
```
import matplotlib.pyplot as plt
for colorMap in plt.colormaps():
    gradient = np.linspace(0, 1, 256)
    gradient = np.vstack((gradient, gradient))

    fig, ax = plt.subplots(figsize=(6, 0.5))
    ax.set_title(colorMap)
    ax.imshow(gradient, aspect='auto', cmap=plt.get_cmap(colorMap))
    ax.set_axis_off()
    print('colorMap=', colorMap)
    plt.show()
    plt.close(fig)
```
